v21.1/cockroach-debug-zip.md

```diff
@@ -38,19 +19,47 @@ The file produced by `cockroach debug zip` can contain highly [sensitive, identi
 
 There are two scenarios in which `debug zip` is useful:
 
-- To collect all of your nodes' logs, which you can then parse to locate issues. It's important to note, though, that `debug zip` can only access logs from active nodes. For more information, see [Collecting log files](#collecting-log-files) below.
+- To collect all of your nodes' logs, which you can then parse to locate issues. You can optionally use the [flags](#flags) to [retrieve only the log files](#generate-a-debug-file-with-logs-only). For more information about logs, see [Logging](logging-overview.html).
+
+	{{site.data.alerts.callout_info}}
+	`cockroach debug zip` can only access logs from active nodes. Nodes not currently connected to the cluster cannot respond to the request, so their log files *are not* included in the `debug zip` output. In such situations, we recommend using the [`--host` flag](#general) to point `debug zip` at individual nodes until data has been gathered for the entire cluster.
```

Comment:
There's some confusion in this paragraph. There are *two separate problems*, which need two separate guidelines:

- nodes that are currently down cannot deliver their logs over the network at all. For these nodes, the operator must log onto the machine where the `cockroach` process would be running otherwise, and gather the files by hand.

- nodes that are currently up but *disconnected from other nodes* (e.g. because of a network partition) may not be able to respond to `debug zip` requests *forwarded by other nodes*, but can still respond to requests for data when asked directly.